lib/puppet/provider/docker_compose/ruby.rb

```diff
@@ -0,0 +1,29 @@
+Puppet::Type.type(:docker_compose).provide(:ruby) do
+  desc "Support for Docker Compose"
+  commands :dockercompose => "docker-compose"
```

Comment:
The module already supports passing several environment variables (like DOCKER_HOST) in via params. Ideally we'd need to support these (or at least the most common ones) for the compose CLI too.